base/timing.jl

```diff
@@ -207,6 +207,12 @@ macro time(ex)
         local stats = gc_num()
         local compile_elapsedtime = cumulative_compile_time_ns_before()
```

Comment:
I think the order of this and the next line should be swapped, since compile time should be a subset of total time. (same at the end of the block)

Reply:
I agree but I didn't do that originally because adding the compile time lookups would add a small amount of time to the time report, and would make small duration things look slower. Maybe it's insignificant and people should be using `@btime` for very fast things?

Reply:
I've made this change. It does seem like the right thing to do and the overhead seems too small to register

Master
```
julia> @time rand();
  0.000000 seconds
```

This PR
```
julia> @time rand()
  0.000000 seconds
```